deploy/crds/cloudingress.managed.openshift.io_publishingstrategies_crd.yaml

```diff
@@ -0,0 +1,137 @@
+              items:
+                description: ApplicationIngress defines application ingress
+                properties:
+                  certificate:
+                    description: SecretReference represents a Secret Reference. It
+                      has enough information to retrieve secret in any namespace
+                    properties:
+                      name:
+                        description: Name is unique within a namespace to reference
+                          a secret resource.
+                        type: string
+                      namespace:
+                        description: Namespace defines the space within which the
+                          secret name must be unique.
+                        type: string
+                    type: object
+                  default:
+                    description: Default defines default value of ingress when cluster
+                      installs
+                    type: boolean
+                  dnsName:
+                    type: string
+                  listening:
+                    description: Listening defines application ingress as internal
+                      or external
+                    type: string
+                  routeSelector:
+                    description: A label selector is a label query over a set of resources.
+                      The result of matchLabels and matchExpressions are ANDed. An
+                      empty label selector matches all objects. A null label selector
+                      matches no objects.
+                    properties:
+                      matchExpressions:
+                        description: matchExpressions is a list of label selector
+                          requirements. The requirements are ANDed.
+                        items:
+                          description: A label selector requirement is a selector
+                            that contains values, a key, and an operator that relates
+                            the key and values.
+                          properties:
+                            key:
+                              description: key is the label key that the selector
+                                applies to.
+                              type: string
+                            operator:
+                              description: operator represents a key's relationship
+                                to a set of values. Valid operators are In, NotIn,
+                                Exists and DoesNotExist.
+                              type: string
+                            values:
+                              description: values is an array of string values. If
+                                the operator is In or NotIn, the values array must
+                                be non-empty. If the operator is Exists or DoesNotExist,
+                                the values array must be empty. This array is replaced
```

Comment:
s/must/may ?

Reply:
this is auto-generated from spec file, does it make sense to change it? @csheremeta @lisa

Reply:
hm, I'm not sure. @lisa do you have an opinion?

Reply:
I'd regenerate. It's important to do without typos since they will be used to auto-generate docs.

Reply:
Will do.